src/Components/WebView/WebView/src/blazor.modules.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
I'm curious about what's going on here. Could you explain?

Reply:
There is no manifest file when there are no modules, so we embed an empty one that gets served as a fallback (same way as how the `_framework/blazor.webview.js`). The alternative (if we want to avoid 404s on the browser) is to special case this path in the StaticContentProvider (which I wanted to avoid because it seemed more hassle).

Reply:
This is in essence the Webview equivalent of the "blazor server endpoint"